recipes/piawka/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "piawka" %}
+{% set version = "0.7.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ../

```

Comment:
This is unusual, can you explain why this is needed?

Reply:
I think I used it for test builds before the static link was there, removing now